src/native/fs/fs_napi.cpp

```diff
@@ -810,25 +810,38 @@ Napi::Value DirWrap::read(const Napi::CallbackInfo& info)
     return api<DirReadEntry>(info);
 }
 
+static Napi::Value
+ISDIR(const Napi::CallbackInfo& info)
```

Comment:
Instead of making a native call every time we want to check a mode,
can we export S_IFMT and S_IFDIR to js land?
Are these constants always defined (perhaps with different values) in Linux and Mac?

Reply:
Like node is doing:
https://github.com/nodejs/node/blob/baa9e652142d0fa02ef21f561ddcdfda6372305f/lib/internal/fs/utils.js#L393-L395
https://github.com/nodejs/node/blob/baa9e652142d0fa02ef21f561ddcdfda6372305f/lib/internal/fs/utils.js#L466-L472